packages/dev/babel-plugin-intl-optim/index.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const {declare} = require('@babel/helper-plugin-utils');
+const glob = require('glob');
+const micromatch = require('micromatch');
+const Path = require('path');
+const template = require('@babel/template').default;
+const {types} = require('@babel/core');
+const t = types;
+
+const requireTemplate = template('INTEROP(require(FILE)).default');
+
+function getContents(filePath, ref) {
+  const resolvedPath = Path.join(filePath, '..', ref);
+  return require(resolvedPath);
```

Comment:
would need an alternate strategy here to avoid NPM cache (either via removing from cache, or ?)